go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/kubevirt/cluster-network-addons-operator
 
-go 1.22
+go 1.23.0
```

Comment:
we cant do it on release-0.95

Reply:
suggestion
https://github.com/kubevirt/cluster-network-addons-operator/pull/2120